activestorage/test/models/variant_test.rb

```diff
@@ -33,7 +33,7 @@ class ActiveStorage::VariantTest < ActiveSupport::TestCase
 
   test "resized and monochrome variation of JPEG blob" do
     blob = create_file_blob(filename: "racecar.jpg")
-    variant = blob.variant(resize_to_limit: [100, 100], colourspace: "b-w").processed
+    variant = blob.variant(resize_to_limit: [100, 100], colorspace: "Gray").processed
```

Comment:
This change was because I did not load defaults, and instead uses mini_magick for the variant processor which results in this test failing. (e.g. `config.active_storage.variant_processor = :vips` fixes it).

However, I will restore the load_defaults as the dummy app did, and reduce the diff. Leaving this here for future reference.